macros/src/actix.rs

```diff
@@ -1491,7 +1493,10 @@ fn handle_unnamed_field_struct(
                 continue;
             }
 
-            let ty_ref = get_field_type(field);
+            let ty_ref = match get_field_type(field) {
```

Comment:
good place for `let-else` if that's allowed by the msrv

Reply:
hmm seems that it depends on the imported crates
With my lockfile it's 1.60 but if I delete it it's 1.65 due to:

┌─────────────────────────────────────────────────────────────────────
│ error: package `hashbrown v0.14.5` cannot be built because it requires rustc 1.63.0 or newer, while the currently active rustc version is 1.62.1                                                                                         │
│ error: package `serde_yaml v0.9.34+deprecated` cannot be built because it requires rustc 1.64 or newer, while the currently active rustc version is 1.63.0                                                                               │
│ error: package `regex-automata v0.4.7` cannot be built because it requires rustc 1.65 or newer, while the currently active rustc version is 1.64.0                                                                                       │
└─────────────────────────────────────────────────────────────────────
```

Reply:
Fair enough. The MSRV aware resolver can't come soon enough